alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/action/DeleteDestinationRequest.kt

```diff
@@ -22,19 +22,19 @@ import org.elasticsearch.common.io.stream.StreamInput
 import org.elasticsearch.common.io.stream.StreamOutput
 import java.io.IOException
 
-class DeleteDestinationRequest() : ActionRequest() {
+class DeleteDestinationRequest : ActionRequest {
 
-    var destinationID: String? = null
-    var refreshPolicy: WriteRequest.RefreshPolicy? = null
+    val destinationId: String
```

Comment:
NP: I believe the old naming scheme was more appropriate since we typically only [begin lowercasing acronyms when they exceed two letters](https://kotlinlang.org/docs/reference/coding-conventions.html#choosing-good-names).

This is not a blocker for this PR, we can always change it in another one of the API PRs.